README.md

```diff
@@ -3,16 +3,16 @@
 ## Run locally in Docker
 
 1. Copy `.env.local.template` to `.env.local` and optionally edit values
-1. Run `docker-compose up` to start the app locally
+1. Run `docker-compose --file ./docker-compose.yml up` to start the app locally
 1. Browse to http://localhost:3000
 
-If you pull down changes, you'll need to run `docker-compose build` to build them and then `docker-compose up` again.
+If you pull down changes, you'll need to run `docker-compose --file ./docker-compose.yml build` to build them and then `docker-compose --file ./docker-compose.yml up` again.
 
 ## Local development
 
 1. Copy `.env.local.template` to `.env.local` and optionally edit values
-1. Install dependencies, `yarn install`
-1. Run `yarn dev` to start the local development server
+1. Install dependencies, `npm install` or `yarn install`
+1. Run `npm run dev` or `yarn dev` to start the local development server
```

Comment:
IMO, we should pick one. Otherwise we should really be maintaining lock files for both `npm` and `yarn`.